ppx/jsx-ppx/tyxml_jsx.ml

```diff
@@ -5,6 +5,7 @@ open Asttypes
 open Tyxml_ppx_common
 
 let make_html_attr_name name =
+  (* Aria attributes are the same as reason-react https://github.com/reasonml/reason-react/blob/5ce8498207ca1f2e9b8305cfca231c837b2d021b/src/ReactDOMRe.re *)
```

Comment:
This treatement of aria is not good, and not in line with what we do in tyxml anyway. Instead, use a general transform that turns everything of the form `aria*` into `aria-lowercase(*)`.